databases/week3/Menu.sql

```diff
@@ -0,0 +1,462 @@
+-- Create database
+CREATE DATABASE DBweek3 DEFAULT CHARACTER SET = 'utf8mb4';
```

Comment:
Databases created like intended! Maybe a little bit of spacing would make it a little easier to read ;)